src/requirements/Form/components/AddFormButton.tsx

```diff
@@ -74,19 +62,6 @@ const AddFormButton = ({ baseFieldPath, isDisabled }) => {
           }}
         />
       )}
-
-      {step === "SELECT_ROLE" && (
```

Comment:
Unfortunately if the user won't add the form to a role as a reward, they won't be able to use it as a requirement neither. I know, this is a pretty strange behaviour, but this change isn't correct because of that :/

Reply:
Indeed. I have an alternative solution here we should talk about. :wink: